detector/fcal/BeamCal_o1_v01_geo.cpp

```diff
@@ -32,7 +32,12 @@ static DD4hep::Geometry::Ref_t create_detector(DD4hep::Geometry::LCDD& lcdd,
   
   DD4hep::XML::setDetectorTypeFlag( element, sdet ) ;
   
-  if( lcdd.buildType() == DD4hep::BUILD_ENVELOPE ) return sdet ;
+  std::cout << "Build type: " << lcdd.buildType() << std::endl;
+
+  if( lcdd.buildType() == DD4hep::BUILD_ENVELOPE ) { std::cout << "Building envelope.\n"; return sdet ; }
+  else { std::cout << "Building all.\n"; }
```

Comment:
By the way, this is for debugging purposes. I am trying to figure out how to write xml to get lcdd.buildType() == DD4hep::BUILD_ENVELOPE. So far unsuccessful. Appreciate all hints.